_posts/2025-01-15-logdetective-explain.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "Log Detective can now explain build log failures"
+author: "Tomas Tomecek"
+layout: "post"
+---
+
+We're thrilled to announce new functionality of [Log
+Detective](https://logdetective.com/). The service can now explain in detail a
+build log of your choice. Click on
+[logdetective.com/explain](https://logdetective.com/explain), paste a
+link to your log, wait one minute and you should see description of important
+lines with the overall explanation.
+
+
+## Disclaimer
+
+This is our initial prototype of this functionality. We appreciate all your
+feedback. Please, be patient as there will be service disruptions, instability
+and usability issues.
+
+* **1️⃣ The service can process only one request at a time.** We understand this
+is very limitng, especially when multiple people will use the service at the
+same time. You may need to wait several minutes to get a reply in such a case.
+We are thankful to the Fedora Project for sponsoring our infrastructure needs,
+but in order to reach the scale of popular AI services, we would need much more
+GPU compute.
```

Comment:
Is this our limit? Or that we simply do not know how to paralelize access to GPU?

Reply:
I think it's both, even the parallelism feature of llama-cpp-server splits context window (which could be a problem for us) - so the easiest way to scale up is to add more GPUs.

We can definitely add the point about parallelism in the text though I didn't know how to phrase it so it would be easy to digest.